vignettes/tutorial2.Rmd

```diff
@@ -326,41 +325,20 @@ We can extract a single field from each time series and plot them all for compar
 ```{r plotfields, fig.show='hold', eval=TRUE} 
 ## Extract a single example field from each series and create a plot
 if(dofieldplots) {
-plotglobalfieldsT <- function(fieldlist, sequencenum, minval=-3.5, maxval=3.5, legendstr) {
-    ## Extract a single example field from each series (sequencenum determines which one) and create a plot
-    lapply(fieldlist, function(g) {
-        suppressWarnings(
-            plot_field(g[sequencenum, 1:Ngrid], emulator$griddataT, 14,  # 14 is the number of color levels in the plot
-                       minval, maxval) +
-                guides(fill=ggplot2::guide_colorbar(barwidth=ggplot2::unit(4,'in'), barheight=ggplot2::unit(0.125,'in'),
-                                                    title=legendstr, title.position='top', title.hjust=0.5))
-        )
-    })
-}
 
-plotglobalfieldsP <- function(fieldlist, sequencenum, minval=-1.5, maxval= 1.5, legendstr) {
-    ## Extract a single example field from each series (sequencenum determines which one) and create a plot
-    lapply(fieldlist, function(g) {
-        suppressWarnings(
-            plot_field(g[sequencenum, (Ngrid + 1):(2*Ngrid)], emulator$griddataP, 14,  # 14 is the number of color levels in the plot
-                       minval, maxval) +
-                guides(fill=ggplot2::guide_colorbar(barwidth=ggplot2::unit(4,'in'), barheight=ggplot2::unit(0.125,'in'),
-                                                    title=legendstr, title.position='top', title.hjust=0.5))
-        )
-    })
-}
+residfieldplotsT <- plotglobalfieldsT(residgrids[1:nplt], yearindex = exfld, 
```

Comment:
Is it worth adding a comment about how the temp and precip results will be plotted with two different function calls?